<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageView</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/images/logo.png</normaloff>:/images/images/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLabel" name="lab_curframe">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSlider" name="slider_frame">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="PaintBoard" name="gView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>885</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMENU">
    <property name="title">
     <string>MENU</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWINDOW">
    <property name="title">
     <string>WINDOW</string>
    </property>
    <addaction name="actDockerLay"/>
    <addaction name="actB_C"/>
   </widget>
   <widget class="QMenu" name="menuHELP">
    <property name="title">
     <string>HELP</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTOOL">
    <property name="title">
     <string>TOOL</string>
    </property>
   </widget>
   <addaction name="menuMENU"/>
   <addaction name="menuWINDOW"/>
   <addaction name="menuHELP"/>
   <addaction name="menuTOOL"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpenFolder"/>
   <addaction name="actOpenFile"/>
   <addaction name="actCloseImg"/>
   <addaction name="actQuit"/>
   <addaction name="separator"/>
   <addaction name="actZoomIn"/>
   <addaction name="actZoomOut"/>
   <addaction name="actZoomRealSize"/>
   <addaction name="actFit_Width"/>
   <addaction name="actFit_Height"/>
   <addaction name="actFit_Page"/>
  </widget>
  <widget class="QDockWidget" name="dock_layer">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>200</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
QCheckBox::indicator:checked
  {
    image: url(:/images/images/eye.png);
width:20px;height:20px;
size
  }

QCheckBox::indicator:unchecked
  {
    image: url(:/images/images/hide.png);
width:20px;height:20px;
  }</string>
   </property>
   <property name="windowTitle">
    <string>layer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>13</y>
       <width>121</width>
       <height>76</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cb_ori">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>origin image</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_border">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>border</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="actOpenFolder">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/folder.ico</normaloff>:/images/images/folder.ico</iconset>
   </property>
   <property name="text">
    <string>Folder</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/quit.png</normaloff>:/images/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actOpenFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/openfile.png</normaloff>:/images/images/openfile.png</iconset>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actZoomIn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/zoom-in.ico</normaloff>:/images/images/zoom-in.ico</iconset>
   </property>
   <property name="text">
    <string>zoom in</string>
   </property>
   <property name="toolTip">
    <string>放大图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actZoomOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/zoom-out.ico</normaloff>:/images/images/zoom-out.ico</iconset>
   </property>
   <property name="text">
    <string>zoom out</string>
   </property>
   <property name="toolTip">
    <string>缩小图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actZoomRealSize">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/zoom.png</normaloff>:/images/images/zoom.png</iconset>
   </property>
   <property name="text">
    <string>resize</string>
   </property>
   <property name="toolTip">
    <string>图片实际大小显示</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actDockerLay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>layer</string>
   </property>
  </action>
  <action name="actCloseImg">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/cross.png</normaloff>:/images/images/cross.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actFit_Width">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/auto_fit_width_regular_icon_205619.ico</normaloff>:/images/images/auto_fit_width_regular_icon_205619.ico</iconset>
   </property>
   <property name="text">
    <string>Fit_Width</string>
   </property>
  </action>
  <action name="actFit_Height">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/auto_fit_height_regular_icon_205620.ico</normaloff>:/images/images/auto_fit_height_regular_icon_205620.ico</iconset>
   </property>
   <property name="text">
    <string>Fit Height</string>
   </property>
  </action>
  <action name="actFit_Page">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/icons8-fit-to-page-32.png</normaloff>:/images/images/icons8-fit-to-page-32.png</iconset>
   </property>
   <property name="text">
    <string>Fit Page</string>
   </property>
  </action>
  <action name="actB_C">
   <property name="text">
    <string>Bright and Contrast</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintBoard</class>
   <extends>QGraphicsView</extends>
   <header location="global">paintboard.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
